README.md

```diff
@@ -6,12 +6,23 @@ Dashboard builder for Prometheus.
 
 [![Build Status](https://travis-ci.org/prometheus/promdash.svg?branch=master)](https://travis-ci.org/prometheus/promdash)
 
-    # Ruby tests.
-    bundle exec rake
-
-    # Javascript tests.
-    RAILS_ENV=test bundle exec rake spec:javascript
-
+### Ruby
+```bash
```

Comment:
whitespace nit: for making things easier to read (especially in the JSHint section), please put an empty line before and after any code blocks.

We should also be consistent in general with empty lines after headers, though either way is fine, given that we have both variants in this README already.